pints/__init__.py

```diff
@@ -227,11 +227,12 @@ def version(formatted=False):
 from ._mcmc._slice_stepout import SliceStepoutMCMC
 from ._mcmc._summary import MCMCSummary
 
-if sys.hexversion >= 0x03030000:
-    from ._mcmc._nuts import NoUTurnMCMC
-else:   # pragma: no python 3 cover
-    import warnings
-    warnings.warn('No-U-Turn sampler unsupported for Python version < 3.3')
+# We may want to move to the newer asyncio and exclude older Pythons for NUTS?
+#if sys.hexversion >= 0x03030000:
+#    from ._mcmc._nuts import NoUTurnMCMC
+#else:   # pragma: no cover
+#    import warnings
+#    warnings.warn('No-U-Turn sampler unsupported for Python version < 3.3')
```

Comment:
Can these lines be rationalised?

Reply:
They're no longer needed, but we will probably need to update this soon to use the newer syntax for asyncio and coroutines etc., so then we'll need to stick something like this back in again, I think.

Happy to remove